packages/@aws-cdk/cdk/lib/app.ts

```diff
@@ -29,29 +29,48 @@ export interface AppProps {
 
   /**
    * Include stack traces in construct metadata entries.
-   * @default true stack traces are included
+   * @default true stack traces are included unless `aws:cdk:disable-stack-trace` is set in the context.
    */
   readonly stackTraces?: boolean;
 
   /**
    * Include runtime versioning information in cloud assembly manifest
-   * @default true runtime info is included
+   * @default true runtime info is included unless `aws:cdk:disable-runtime-info` is set in the context.
    */
   readonly runtimeInfo?: boolean;
 
   /**
-   * Additional context values for the application
+   * Additional context values for the application.
    *
-   * @default No additional context
+   * Context can be read from any construct using `node.getContext(key)`.
+   *
+   * @default - no additional context
    */
   readonly context?: { [key: string]: string };
 }
 
 /**
- * Represents a CDK program.
+ * A construct which represents an entire CDK app. This construct is normally
+ * the root of the construct tree.
+ *
+ * You would normally define an `App` instance in your program's entrypoint,
+ * then define constructs where the app is used as the parent scope.
+ *
+ * After all the child constructs are defined within the app, you should call
+ * `app.synth()` which will emit a "cloud assembly" from this app into the
```

Comment:
This not `run`? I'm confused, because the doc of `run` says it'll return a "cloud assembly" 🤷🏻‍♂️